.env

```
@@ -2,7 +2,7 @@ APP_NAME="SINF Website 2023"
 APP_ENV=local # Use "maintenance" to enable the maitenance page
 APP_KEY=base64:AVegBkyjHs2QQsx53M9Kt9GbnrDLc6yG4cRg+WMJrbA=
 APP_DEBUG=true
-APP_URL=http://localhost
+APP_URL=http://localhost:8000/
```

The slash at the end of this can cause problems

```suggestion
APP_URL=http://localhost:8000
```